Automattic/wp-calypso
# Untangling: sync GSV tabs with URL

Related to:

- https://github.com/Automattic/dotcom-forge/issues/6725
- https://github.com/Automattic/dotcom-forge/issues/6747
- https://github.com/Automattic/dotcom-forge/issues/6748
- https://github.com/Automattic/dotcom-forge/issues/6781
- https://github.com/Automattic/dotcom-forge/issues/6777

## Proposed Changes

This PR updates the URL with the corresponding GSV tab's URL whenever it is clicked.

## Note

Let's ignore any weird stylings and focus on functionality. We should merge this ASAP to unblock others.

## Testing Instructions

1. Go to `<Calypso Live URL>/sites?flags=layout/dotcom-nav-redesign-v2`
2. Search a site
3. Click on a site
4. Verify that the preview pane is opened
5. Try clicking the tabs around
6. Verify that the URL changes

## Pre-merge Checklist
